ansible/playbooks/aix.yml

```diff
@@ -717,6 +717,30 @@
             # Environment variables don't work as part of command
             - skip_ansible_lint
 
+      #########################
+      # Install Asian Locales #
+      #########################
+        - name: Transfer locales installp to remote host
+          copy:
+            src: /Vendor_Files/aix/asian.tar.gz
+            dest: /tmp/locale/
+          tags: locale
+
+        - name: Extract asian.tar
+          shell: gzip -dc /tmp/locale/asian.tar.gz | tar xf - -C /tmp/locale
+          tags: locale
+
+        - name: Install locales - installp
+          command: installp -aXYgd /tmp/locale all
+          register: result.locale
```

Comment:
What will installp do if this is already installed on the system?

Reply:
The result is success with "already installed" message. I added to check available locales, anyway.